### DISCUSSION TITLE
Kohya can't load caption files

### DISCUSSION THREAD

**Discussion by MrDaisyBates** [bmaltais/kohya_ss#1481] [Q&A]
![image](https://github.com/bmaltais/kohya_ss/assets/143578183/a63c143e-4334-44d8-a8cb-334d64c0a760)
Hey guys. Sorry to bother, but:

I very much so have the text files with the appropriate names in the folder with the training images. I used Kohya blip captioning to create them, as well. Because they weren't working, I deleted them and did it again and still have the same issue. 

I'm using Manjaro Linux if it matters to you. I'm also running with a bit older hardware at this point (i7 4790k, 1080ti).

Any help would be _immensely_ appreciated!

Edit: I'm also NOT very terminal savvy. Feel free to really talk down to me when you give me guidance.

**Comment by MrDaisyBates** [comment]
Problem solved:

How embarrassing. Default setting is to look for .caption files. I needed to specify .txt files in the training parameter settings. I didn't notice that. 

Leaving this here for future internet travelers. 

Wishing you well. <3

**Reply by Vendaciousness** [reply]
Are you going to give us the parameter settings? Because I have the same problem and already knew what you said, but have been looking for the parameter/flag/argument for running from command line.
